src/pages/v2/p/[projectId]/index.page.tsx

```diff
@@ -1,14 +1,20 @@
+import Loading from 'components/Loading'
 import { AppWrapper, SEO } from 'components/common'
+import { FEATURE_FLAGS } from 'constants/featureFlags'
 import { PV_V2 } from 'constants/pv'
 import { V2V3ProjectPageProvider } from 'contexts/v2v3/V2V3ProjectPageProvider'
 import { paginateDepleteProjectsQueryCall } from 'lib/apollo/paginateDepleteProjectsQuery'
+import { ProjectMetadata } from 'models/projectMetadata'
 import { GetStaticPaths, GetStaticProps, InferGetStaticPropsType } from 'next'
+import { Suspense, lazy } from 'react'
+import { featureFlagEnabled } from 'utils/featureFlags'
 import { cidFromUrl, ipfsPublicGatewayUrl } from 'utils/ipfs'
 import {
   ProjectPageProps,
   getProjectStaticProps,
 } from 'utils/server/pages/props'
 import { V2V3Dashboard } from './components/V2V3Dashboard'
+const ProjectDashboard = lazy(() => import('components/ProjectDashboard'))
```

Comment:
dont import new project page code unless ff is enabled. save load time while in dev